source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -229,13 +204,13 @@ void BioXASSideTableCoordinator::onHorizontalFeedbackControlValueChanged()
 
 void BioXASSideTableCoordinator::onHorizontalStatusControlValueChanged()
 {
-	if(!connectedOnce_)
+	if ( !connectedOnce_ )
 		return;
 
 	// Status value: 0 - "MOVE DONE";  1 - "MOVE ACTIVE"; 2 - "AT LIMIT" ("MINOR"); 3 - "FORCED STOP" ("MINOR"); 4 - "ERROR" ("MAJOR")
 	// Status is "MOVE ACTIVE" if at least one motor is moving or the largest status value
 	double status = 0;
-	if (horizontalUpstreamStatusControl_->withinTolerance(1) || horizontalDownstreamStatusControl_->withinTolerance(1)) {
+	if (horizontalUpstreamStatusControl_->isMoving() || horizontalDownstreamStatusControl_->isMoving()) {
```

Comment:
Earlier comment applies here too.  You don't need to address each one, but I don't think that this is correct.